db/migrate/20150714144500_review_discovery_permissions.rb

```diff
@@ -0,0 +1,12 @@
+class ReviewDiscoveryPermissions < ActiveRecord::Migration
+  def up
+    if mgr = Role.find_by_name("Discovery Manager")
+      mgr.add_permissions!(["submit_discovered_hosts", "auto_provision_discovered_hosts", "create_discovery_rules"])
```

Comment:
Good point, this is what I am going to do:

* if `edit_discovered_hosts` is present add `submit_discovered_hosts`
* if `provision_discovered_hosts` is present add `auto_provision_discovered_hosts`
* if `new_discovery_rules` is present add `create_discovery_rules`

Objections?